### PR TITLE
core(scoring): update CLS score curve - 0.25 is now failing

### PR DESCRIPTION
**Pull request by paulirish**
keeping up with the joneses. 

CLS of 0.25 and above is now considered failing
tweak the PODR to make sure CLS 0.10 is still a score of 90/100

https://goto.google.com/tbsgg

### CODE REVIEW

File: lighthouse-core/audits/metrics/cumulative-layout-shift.js [hunk 1]
```diff
@@ -46,9 +46,9 @@ class CumulativeLayoutShift extends Audit {
     return {
       // Calibrated to assure 0.1 gets a score of 0.9. https://web.dev/cls/#what-is-a-good-cls-score
       // This 0.1 target score was determined through both manual evaluation and large-scale analysis.
-      // see https://www.desmos.com/calculator/wmcxn7zfhc
-      scorePODR: 0.02,
-      scoreMedian: 0.2,
+      // see https://www.desmos.com/calculator/vjb652sqeh
```
Comment (patrickhulce) [inline lighthouse-core/audits/metrics/cumulative-layout-shift.js]: seems like it didn't save with the new values, maybe just forgot to press save?

File: lighthouse-core/audits/metrics/cumulative-layout-shift.js [hunk 2]
```diff
@@ -46,9 +46,9 @@ class CumulativeLayoutShift extends Audit {
     return {
       // Calibrated to assure 0.1 gets a score of 0.9. https://web.dev/cls/#what-is-a-good-cls-score
```
Comment (patrickhulce) [inline lighthouse-core/audits/metrics/cumulative-layout-shift.js]: we should really consider reworking our math so we can define in terms of the 90 cutoff :) haha

**Comment by brendankenny** [comment]
lol how does this file have no tests

**Comment by brendankenny** [comment]
> I'm not sure what's here, so if it matters to the review maybe get a googler's eyes too?

it's just a doc of thresholds used for field data too

> > lol how does this file have no tests
> 
> FWIW the sample JSON artifact serves the same function as what I imagine would be the lone unit test

substitute "this file" with "these files" to include the computed artifact too :)

File: lighthouse-core/audits/metrics/cumulative-layout-shift.js (hunk 1 above)
Comment (paulirish) [reply lighthouse-core/audits/metrics/cumulative-layout-shift.js]: image


true. just updated with the newer shiny one.

**Comment by patrickhulce** [comment]
conflicts though :)

**Pull request merged**